Segments/Blackburrow.mapproj

```diff
@@ -787,7 +787,10 @@
         
         public override bool CanEquip(MobileEntity entity)
         {
-            if (entity is PlayerEntity { Profession: Thief })
+        
+            var skilllevel = entity.GetSkillLevel(Skill.Magic);
+            
+            if (entity is PlayerEntity { Profession: Thief } & skilllevel >= 15)
```

Comment:
`I would split these conditionals. Just from clarity standpoint. 

var skilllevel = entity.GetSkillLevel(Skill.Magic);
var isUsable = false;

/* 
Making sure the character is thief and level 15
*/
 if (entity is PlayerEntity player)
            {
                if (player.Profession == Profession.Thief && skilllevel >= 15)
                {
                    isUsable = true;
                }
            }
        }
        
        return isUsable;
               
`